[2024-05-06T04:54:02.584+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T04:54:02.716+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.create_entry_gcs manual__2024-05-06T04:53:55.730459+00:00 [queued]>
[2024-05-06T04:54:02.732+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.create_entry_gcs manual__2024-05-06T04:53:55.730459+00:00 [queued]>
[2024-05-06T04:54:02.732+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-06T04:54:02.768+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): create_entry_gcs> on 2024-05-06 04:53:55.730459+00:00
[2024-05-06T04:54:02.827+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2255) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-06T04:54:02.836+0000] {standard_task_runner.py:63} INFO - Started process 2266 to run task
[2024-05-06T04:54:02.834+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'create_entry_gcs', 'manual__2024-05-06T04:53:55.730459+00:00', '--job-id', '6', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpbi1jyprf']
[2024-05-06T04:54:02.845+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask create_entry_gcs
[2024-05-06T04:54:03.009+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.create_entry_gcs manual__2024-05-06T04:53:55.730459+00:00 [running]> on host 0cea9ba68f60
[2024-05-06T04:54:03.352+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='create_entry_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T04:53:55.730459+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T04:53:55.730459+00:00'
[2024-05-06T04:54:03.354+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T04:54:03.354+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-06T04:54:03.355+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo create_entry_gcs']
[2024-05-06T04:54:03.396+0000] {subprocess.py:86} INFO - Output:
[2024-05-06T04:54:03.400+0000] {subprocess.py:93} INFO - create_entry_gcs
[2024-05-06T04:54:03.401+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-06T04:54:03.402+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T04:54:03.496+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_gcs, execution_date=20240506T045355, start_date=20240506T045402, end_date=20240506T045403
[2024-05-06T04:54:03.592+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-06T04:54:03.656+0000] {taskinstance.py:3482} INFO - 6 downstream tasks scheduled from follow-on schedule check
[2024-05-06T04:54:03.667+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
